packages/engine/src/worker/runner/python/mod.rs

```diff
@@ -156,20 +167,36 @@ async fn _run(
                             })?;
                         (Some(payload), Some(wrapper))
                     }
+                    InboundToRunnerMsgPayload::CancelTask(_) => {
+                        // Don't send to Python process -- unused for now.
+                        // TODO: Remove `continue` when/if a package uses `CancelTask`
+                        //       and it's implemented in Python.
```

Comment:
I tried this with `todo!()`, but this arm is getting hit because we are already sending cancel messages in several places. I think it's easier to keep the `continue` here and mention this specific code section in the task.

Reply:
Would it be too much work to stop sending the cancels at a higher level? If so then can we make that a next up task soon. Because I'd prefer if we just toggled it in one place with a TODO, rather than at loads of leaf nodes down the execution tree